crates/re_types_builder/src/codegen/python.rs

```diff
@@ -692,7 +685,17 @@ fn code_for_struct(
 
             code.push_text(format!("{name}: {typ}"), 1, 4);
 
-            code.push_text(quote_doc_from_docs(docs), 0, 4);
+            // Generating docs for all the fields creates A LOT of visual noise in the API docs.
+            let show_fields_in_docs = false;
+            if show_fields_in_docs {
+                code.push_text(quote_field_docs(field), 0, 4);
+            } else {
+                code.push_text(
```

Comment:
if we make the effort to leave a comment that the doc string isn't there, we might as well put the commented out docstring in here 🤔 - after all someone just went to look at the code now and wants to know more about the field.